VRDR.CLI/Program.cs

```diff
@@ -620,6 +620,20 @@ static int Main(string[] args)
                     Console.WriteLine($"{info.Field, -5} {info.Name,-15} {Truncate(info.Contents, 75), -75}: \"{field + "\"",-80}");
                 }
             }
+            else if (args[0] == "ijebuilder")

```

Comment:
Is there a "help" option or similar documentation explaining what the CLI arg options are? If yes, can this be added and explained there?

Reply:
We should eventually make this a "real" CLI tool with better option handling; for now I'm adding the new one to the README (along with some others that have been missing)